app/src/androidTest/java/com/github/se/cyrcle/ui/testEnd2End/MainActivityTest.kt

```diff
@@ -62,6 +89,85 @@ class MainActivityTest {
     cardRobot.assertParkingDetailsScreen()
   }
 
+  // ============================================================================
+  // ============================ ADDPARKINGS ROBOT =============================
+  // ============================================================================
+  private class AddParkingRobot(val composeTestRule: ComposeTestRule) {
+
+    fun assertLocationPickerScreen() {
+      composeTestRule.onNodeWithTag("LocationPickerScreen").assertIsDisplayed()
+      composeTestRule.onNodeWithTag("LocationPickerBottomBar").assertIsDisplayed()
+      composeTestRule.onNodeWithTag("LocationPickerTopBar").assertIsDisplayed()
+      composeTestRule.onNodeWithTag("cancelButton").assertIsDisplayed().assertHasClickAction()
+      composeTestRule.onNodeWithTag("nextButton").assertIsDisplayed().assertHasClickAction()
+    }
+
+    @OptIn(ExperimentalTestApi::class)
+    fun selectLocation() {
+      composeTestRule.onNodeWithTag("nextButton").assertHasClickAction().performClick()
+
+      composeTestRule.waitUntilAtLeastOneExists(hasTestTag("toggleRectangleButton"))
+    }
+
+    @OptIn(ExperimentalTestApi::class)
+    fun makeRectangleAndNext() = runTest {
+      composeTestRule.onNodeWithTag("LocationPickerScreen").performTouchInput {
+        swipe(start = Offset(0.5f, 0.5f), end = Offset(0.7f, 0.7f), durationMillis = 10)
```

Comment:
Not realy, just to make a lovely rectangle xD

Reply:
Sounds good, but is it compatible with different screen types? Using values that depend on the screen size is usually to be preferred in these cases no? I just wanna make sure so that we don't wonder why the CI fails down the line...